File: 兴趣班入门百练讲义/NQ100/NQ024/NQ024_improved.cpp
// NQ024 珠穆朗玛峰测距
#include <cmath>
#include <cstdio>
int main() {
    int c;
    double x,y,m,n;
    scanf("%d",&c);
    while(c--){
        scanf("%lf %lf %lf %lf",&x,&y,&m,&n);
        double d=sqrt((x-m)*(x-m)+(y-n)*(y-n));  // 计算欧几里得距离
        printf("%.1lf\n",d);  // 输出保留一位小数
    }
    return 0;
}

File: source/algorithm/NQ024/NQ024_improved.py
# NQ024 珠穆朗玛峰测距
import math
def main():
    c=int(input())
    for _ in range(c):
        x,y,m,n=map(float,input().split())
        d=math.sqrt((x-m)**2+(y-n)**2)  # 计算欧几里得距离
        print("{0:.1f}".format(d))  # 输出保留一位小数
if __name__=="__main__":
    main()